packages/perseus/src/widgets/interactive-graphs/graphs/components/hairlines.tsx

```diff
@@ -0,0 +1,42 @@
+import {color} from "@khanacademy/wonder-blocks-tokens";
```

Comment:
LOVE that this is a separate file, this makes way more sense while also making this code far easier to locate.